modules/proxy/proxy_util.c

```diff
@@ -815,7 +815,8 @@ PROXY_DECLARE(int) ap_proxy_checkproxyblock(request_rec *r, proxy_server_conf *c
 PROXY_DECLARE(int) ap_proxy_pre_http_request(conn_rec *c, request_rec *r)
 {
     ap_add_input_filter("HTTP_IN", NULL, r, c);
-    return OK;
+    ap_add_input_filter("HTTP1_BODY_IN", NULL, r, c);
```

Comment:
Adding this for proxy request. Could check on protocol version, but this only runs on HTTP/1.x requests.